core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -449,6 +449,20 @@ private String getNextTasksToBeScheduled(Workflow workflow, Task task) {
                 // Reset integer overflow to max value
                 startDelay = retryDelaySeconds < 0 ? Integer.MAX_VALUE : retryDelaySeconds;
                 break;
+            case CUSTOM:
```

Comment:
Following the comment above, the switch statement here could check for the retry logic override within workflow task and then the task definition if not specified.

Reply:
Done.